src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -2341,7 +2343,7 @@
 		01C6AC2D252B23FC0052814D /* QRScanner */ = {
 			isa = PBXGroup;
 			children = (
-				01C6AC25252B23D70052814D /* ExposureSubmissionQRScannerFocusView.swift */,
+				01C6AC25252B23D70052814D /* QRScannerFocusView.swift */,
```

Comment:
if we use the QRScannerFocus View in two places wo should place the file into a shared section.
I would also prefer a version without a xib file

Reply:
the QRScannerFocusView doesn't have a nib file, i will move the file to a more general location

Reply:
it gets referenced by the xib you created / copied.